### ISSUE TITLE
favicon.ico error

### ISSUE THREAD

**Issue by sebastian-fox** [ukgovdatascience/govdown#10]
Hi @nacnudus - love the package. 
I'm getting an error when trying to run the template without amending it.  Something to do with assets/images/favicon.ico:
```
processing file: govdown.Rmd
  |......................                                           |  33%
  ordinary text without R code

  |...........................................                      |  67%
label: unnamed-chunk-1
  |.................................................................| 100%
  ordinary text without R code


output file: govdown.knit.md

"C:/PROGRA~1/Pandoc/pandoc" +RTS -K512m -RTS govdown.utf8.md --to html4 --from markdown+autolink_bare_uris+ascii_identifiers+tex_math_single_backslash-implicit_figures+smart --output govdown.html --email-obfuscation none --self-contained --include-in-header "C:\Users\sebastian.fox\Documents\R\R-3.5.3\library\govdown\rmarkdown\resources\favicon.html" --css "C:\Users\sebastian.fox\Documents\R\R-3.5.3\library\govdown\rmarkdown\resources\govukish.css" --standalone --self-contained --template "C:/Users/sebastian.fox/Documents/R/R-3.5.3/library/govdown/rmarkdown/resources/govuk.html" --lua-filter "C:/Users/sebastian.fox/Documents/R/R-3.5.3/library/govdown/rmarkdown/resources/govuk.lua" --resource-path ".:C:/Users/sebastian.fox/Documents/R/R-3.5.3/library/govdown/rmarkdown/resources" --highlight-style=pygments --mathjax --include-in-header "C:\Users\SEBAST~1.FOX\AppData\Local\Temp\RtmpQ9yz6e\rmarkdown-str9284b1e517f.html" --include-before-body "C:\Users\SEBAST~1.FOX\AppData\Local\Temp\RtmpQ9yz6e\rmarkdown-str9286d775d0.html" 
File assets/images/favicon.ico not found in resource path
Error: pandoc document conversion failed with error 99
Execution halted
```

**Comment by sebastian-fox** [comment]
```
> sessioninfo::session_info()
- Session info -----------------------------------------------------------------------------------------------
 setting  value                       
 version  R version 3.5.3 (2019-03-11)
 os       Windows >= 8 x64            
 system   x86_64, mingw32             
 ui       RStudio                     
 language (EN)                        
 collate  English_United Kingdom.1252 
 ctype    English_United Kingdom.1252 
 tz       Europe/London               
 date     2019-04-05                  

- Packages ---------------------------------------------------------------------------------------------------
 package     * version    date       lib source                                   
 assertthat    0.2.1      2019-03-21 [1] CRAN (R 3.5.3)                           
 backports     1.1.3      2018-12-14 [1] CRAN (R 3.5.2)                           
 callr         3.2.0      2019-03-15 [1] CRAN (R 3.5.3)                           
 cli           1.1.0      2019-03-19 [1] CRAN (R 3.5.3)                           
 crayon        1.3.4      2017-09-16 [1] CRAN (R 3.5.0)                           
 curl          3.3        2019-01-10 [1] CRAN (R 3.5.2)                           
 desc          1.2.0      2018-05-01 [1] CRAN (R 3.5.1)                           
 devtools      2.0.1      2018-10-26 [1] CRAN (R 3.5.2)                           
 digest        0.6.18     2018-10-10 [1] CRAN (R 3.5.1)                           
 evaluate      0.13       2019-02-12 [1] CRAN (R 3.5.2)                           
 fs            1.2.6      2018-08-23 [1] CRAN (R 3.5.1)                           
 glue          1.3.1      2019-03-12 [1] CRAN (R 3.5.3)                           
 govdown       0.0.0.9000 2019-04-05 [1] Github (ukgovdatascience/govdown@fbb8c1a)
 htmltools     0.3.6      2017-04-28 [1] CRAN (R 3.5.0)                           
 knitr         1.22       2019-03-08 [1] CRAN (R 3.5.3)                           
 magrittr      1.5        2014-11-22 [1] CRAN (R 3.5.0)                           
 memoise       1.1.0      2017-04-21 [1] CRAN (R 3.5.0)                           
 pkgbuild      1.0.2      2018-10-16 [1] CRAN (R 3.5.1)                           
 pkgload       1.0.2      2018-10-29 [1] CRAN (R 3.5.1)                           
 prettyunits   1.0.2      2015-07-13 [1] CRAN (R 3.5.0)                           
 processx      3.3.0      2019-03-10 [1] CRAN (R 3.5.3)                           
 ps            1.3.0      2018-12-21 [1] CRAN (R 3.5.2)                           
 R6            2.4.0      2019-02-14 [1] CRAN (R 3.5.2)                           
 Rcpp          1.0.1      2019-03-17 [1] CRAN (R 3.5.3)                           
 remotes       2.0.2      2018-10-30 [1] CRAN (R 3.5.1)                           
 rlang         0.3.3      2019-03-29 [1] CRAN (R 3.5.3)                           
 rmarkdown     1.12       2019-03-14 [1] CRAN (R 3.5.3)                           
 rprojroot     1.3-2      2018-01-03 [1] CRAN (R 3.5.0)                           
 rstudioapi    0.10       2019-03-19 [1] CRAN (R 3.5.3)                           
 sessioninfo   1.1.1      2018-11-05 [1] CRAN (R 3.5.1)                           
 testthat      2.0.1      2018-10-13 [1] CRAN (R 3.5.1)                           
 usethis       1.4.0      2018-08-14 [1] CRAN (R 3.5.1)                           
 withr         2.1.2      2018-03-15 [1] CRAN (R 3.5.0)                           
 xfun          0.6        2019-04-02 [1] CRAN (R 3.5.3)                           
 yaml          2.2.0      2018-07-25 [1] CRAN (R 3.5.1)                           

[1] C:/Users/sebastian.fox/Documents/R/R-3.5.3/library
```

**Comment by nacnudus** [comment]
Confirmed on Windows. Thanks for reporting.

**Comment by nacnudus** [comment]
Fixed in ac0c087ec2a74eeb914c32f61ebc12288e1d30b8.  It was a path-separator thing, see https://pandoc.org/MANUAL.html#option--resource-path.

**Comment by sebastian-fox** [comment]
Thanks @nacnudus - I can confirm it works for me too. I'll check out how you've done it
